test/not_found_handler_test.cc

```diff
@@ -0,0 +1,23 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "not_found_handler.h"
+
+class MockRequest : public Request {
```

Comment:
If we're not going to be calling any of the functions, do we need to use a mock?